src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -113,16 +148,16 @@ public Errors validatePartial(OverseasEntitySubmissionDto overseasEntitySubmissi
                     loggingContext);
         }
 
-        var entityDto = overseasEntitySubmissionDto.getEntity();
-        if (Objects.nonNull(entityDto)) {
-            entityDtoValidator.validate(entityDto, errors, loggingContext);
+        if (!isRoeUpdateEnabled) {
```

Comment:
We should not be checking the flag in the common part - move to registration or update if different.